spec/controllers/api/v1/orders_packages_controller_spec.rb

```diff
@@ -30,6 +30,15 @@
       get :index, search_by_package_id: package.id
       expect( subject["orders_packages"].size ).to eq(3)
     end
+
+    it "return orders_packages of order_id mentioned in params" do
+      order = create :order
+      order2 = create :order
+      create_list(:orders_package, 2, order_id: order.id)
+      create_list(:orders_package, 2,order_id: order2.id)
+      get :index, order_id: order.id
+      expect( subject["orders_packages"].size ).to eq(2)
```

Comment:
you probably want to test that the `orders_packages` returned are the correct ones.
Having 2 records returned doesn't mean much, they could be the wrong ones